doc/source/cluster/cluster_under_construction/key-concepts.rst

```diff
@@ -1,130 +1,97 @@
-.. warning::
-    This page is under construction!
-
-Key Concepts
-============
-..
-    TODO(cade) Can we simplify this? From https://github.com/ray-project/ray/pull/26754#issuecomment-1192927645:
-    * Worker Nodes
-    * Head Node
-    * Autoscaler
-    * Clients and Jobs
-    
-    Need to add the following sections + break out existing content into them.
-    See ray-core/user-guide.rst for a TOC example
-    
-    overview
-    high-level-architecture
-    jobs
-    nodes-vs-workers
-    scheduling-and-autoscaling
-    configuration
-    Things-to-know
-
 .. include:: /_includes/clusters/we_are_hiring.rst
 
-.. _cluster-key-concepts-under-construction:
-
 Key Concepts
 ============
 
-Cluster
--------
+.. _cluster-key-concepts-under-construction:
 
-A Ray cluster is a set of one or more nodes that are running Ray and share the
-same :ref:`head node<cluster-node-types>`.
 
-.. _cluster-node-types-under-construction:
+This page introduces the following key concepts concerning Ray clusters:
 
-Node types
-----------
+.. contents::
+    :local:
 
-A Ray cluster consists of a :ref:`head node<cluster-head-node>` and a set of
-:ref:`worker nodes<cluster-worker-node>`.
+Ray cluster
+------------
+A **Ray cluster** is comprised of a :ref:`head node<cluster-head-node-under-construction>`
+and any number of :ref:`worker nodes<cluster-worker-nodes-under-construction>`.
 
-.. image:: ray-cluster.jpg
+.. figure:: images/ray-cluster.svg
     :align: center
     :width: 600px
+    
+    *A Ray cluster with two worker nodes. Each node runs Ray helper processes to
+    facilitate distributed scheduling and memory management. The head node runs
+    additional helper processes, which are highlighted.*
 
-.. _cluster-head-node-under-construction:
-
-Head node
-~~~~~~~~~
+The number of worker nodes in a cluster may change with application demand, according
+to your Ray cluster configuration. This is known as *autoscaling*. The head node runs
+the :ref:`autoscaler<cluster-autoscaler-under-construction>`.
 
-The head node is the first node started by the
-:ref:`Ray cluster launcher<cluster-launcher>` when trying to launch a Ray
-cluster. Among other things, the head node holds the :ref:`Global Control Store
-(GCS)<memory>` and runs the :ref:`autoscaler<cluster-autoscaler>`. Once the head
-node is started, it will be responsible for launching any additional
-:ref:`worker nodes<cluster-worker-node>`. The head node itself will also execute
-tasks and actors to utilize its capacity.
+.. note::
+    Nodes are implemented as pods when :ref:`running on Kubernetes<kuberay-index>`.
 
-.. _cluster-worker-node-under-construction:
+Users can submit jobs for execution on the Ray cluster, or can interactively use the
+cluster by connecting to the head node and running `ray.init`. See
```

Comment:
Could be even more specific here and include the exact function call: `ray.init()`.

Reply:
This feels like a lot of detail for the introduction of the Key Concepts page. How about I link to the below section `Clients and jobs` and provide this detail? FYI still incorporating Stephanie's feedback for that section.

Reply:
The amount of added detail I'm asking for is the parens.

Reply:
Oh, I guess `connecting to the head node` is a little vague (might be referring generally to Ray Client or SSHing)
Adding detail elsewhere is fine. Don't mind me.